sdk/webpubsub/azure-messaging-webpubsub-client/src/test/java/module-info.java

```diff
@@ -1,31 +0,0 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
webpubsub's module-info java files are deleted? Is this related to the maven surefilre plugin version upgrade?

Reply:
These are test code module-infos which we don't really use and cause a lot of problems with build systems and IDEs. And yeah, this was removed due to the compiler plugin changes

Reply:
LGTM. Not sure whether they are the reason that before the coverage part of this 2 lib is empty
https://github.com/Azure/azure-sdk-for-java/pull/38335#issuecomment-1899763603